lib/front_desk.rb

```diff
@@ -0,0 +1,187 @@
+require_relative "reservation"
+require_relative "block_room"
+
+module Hotel
+  class FrontDesk
+    # attr_reader :all_rooms, :reservations_record, :block_reservations_only, :reserved_rooms_in_blocks
+    attr_accessor :all_rooms, :reservations_record, :block_reservations_only, :reserved_rooms_in_blocks
+
+    def initialize
+      @all_rooms = [*(1..20)] # => makes array with elements from 1 to 20
+
+      @reservations_record = Array.new # => stores all reservation objects
+
+      @block_reservations_only = Array.new # Stores all block reservations
+
+      @reserved_rooms_in_blocks = Array.new # stores the reservations of rooms reserved in blocks
+    end
+
+    def all_rooms_array
+      return @all_rooms
+    end
```

Comment:
I don't think this method is wrong-- in fact, it is the perfect example of a "getter method"! However, since what you are returning is an instance variable (`@all_rooms`), there are ways to refactor this to be shorter and smaller. Think on it and let me know what questions you have :)